src/shcKeyValidator.ts

```diff
@@ -152,20 +151,20 @@ function validateX5c(x5c: string[], log: Log): CertFields | undefined {
             subjectAltName: subjectAltName
         }
     } catch (err) {
-        log.error('Error validating x5c certificates: ' + err, ErrorCode.INVALID_KEY_X5C);
+        log.error('Error validating x5c certificates: ' + (err as Error).toString(), ErrorCode.INVALID_KEY_X5C);
     } finally {
         certFiles.map((file) => {
             fs.unlinkSync(file);
         })
     }
 }
 
-export async function verifyAndImportHealthCardIssuerKey(keySet: KeySet, log = new Log('Validate Key-Set'), expectedSubjectAltName = ''): Promise<ValidationResult> {
+export async function verifyAndImportHealthCardIssuerKey(keySet: KeySet, log = new Log('Validate Key-Set'), expectedSubjectAltName = ''): Promise<{keySet : KeySet | undefined, log : Log}> {
```

Comment:
Can we avoid returning the keyset here, to be consistent with the other changes. I briefly looked and I don't think we end up using it, or if we do, the caller can simply use the one passed in.